x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.scss

```diff
@@ -0,0 +1,5 @@
+.lnsIndexPatternDimensionEditor__dateHistogramHelpPopover {
+  max-height: 500px;
+  max-width: 400px;
+  overflow-y: scroll;
+}
```

Comment:
I think there's something off about this styling, it scrolls horizontally too:

<img width="412" alt="Screen Shot 2020-12-23 at 11 06 28 AM" src="https://user-images.githubusercontent.com/666475/103015756-fcea3c00-450e-11eb-8ce9-502713a30dbd.png">